locales/en/cos-ui.json

```diff
@@ -86,7 +86,7 @@
   "createConnector": "Create Connector",
   "createConnectorsInstance": "Create Connectors instance",
   "created": "Created",
-  "createOSDTrail": "Create a namespace on an OSD trial cluster",
+  "createOSDTrial": "Create a namespace on an OpenShift Dedicated trial cluster",
```

Comment:
As per @jgiardino we wanted to use "Create a namespace on an OSD trial cluster" only

Reply:
After some discussion we learned yesterday that OSD is not an accepted product acronym and that we need to use at the very least OpenShift Dedicated.